### DISCUSSION TITLE
Choose a name for this new Feature: "Blueprint"

### DISCUSSION THREAD

**Discussion by negue** [negue/meme-box#456] [Polls]
Since 2 Months I'm working on the Feature "Blueprint" (#439), which will be doing that:

- a UI which has a list of "steps" you want to execute
- each step can be awaited, to create more dynamic "anything" (trigger an image here, while that play video and so on)
- it has also support for obs commands
- (maybe not important for the name search, but its based on the scripting backend, so its creating script code from the steps shown in the UI - and with the UI way easier to create this than with just creating a script code itself)

While developing this, the first name was "logic step ui" which sounded meh, right after I thought about naming it "blueprint"

But blueprints usually more for the "building something" or "something will be created if you follow that blueprint" way, which here something not apply.

Then I thought about using the name workflow. Other users also threw in calling it "recipes".

And thats why I need somekind of poll to figure out how to name it and how not 😁 

What we also need is a way to name those steps, since there will be often more and more new steps to choose from.

The Poll will have the new Features Name, but I would love to get a suggestion on how to name the "step" part, so comment down below.

If there are more votes for the "Other, comment here please" we'll redo this.

*Don't forget to like and subscribe and hit that bell button, Kappa*

**Comment by cmgriffing** [comment]
Itinerary?

**Reply by negue** [reply]
I had to search it first :D and then a day later I somehow saw that word again out of nowhere - quite funny :D

**Comment by gigili** [comment]
Action block?

**Reply by negue** [reply]
Action Block as the Big Feature or for the "Step" to be named ?

**Reply by gigili** [reply]
maybe a better name for the whole thing, than just the step, so like each "blueprint" as it's called now would be renamed to action block since it's just a block of actions  to be executed or command blocks has a similar ring to it as well

**Reply by negue** [reply]
action block might be something too - true - if you all want that - please vote this up :>

as for ... "command block" - might be a better name than "step" regardless of the feature name

**Comment by negue** [comment]
Going with the `Recipe` / `Command Blocks` Naming
